GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -127,13 +127,30 @@ Profile
     relax_message = $ON_VESSEL$KERBAL is breathing again
   }
 
+  Rule
+  {
+    name = humidity
+    input = MoistAtmosphere
+	output = MoistAtmosphere
+    rate = 0.002879122904 // 50Ltr per-day @100% humidity contains 850 mL of Water per-day per Kerbal, (humans breath is at 100% humidity)
+    output_only = true // monitor input do not consume it 
```

Comment:
Make a note maybe the assumption is here that kerbals will burn fat reserves when not getting food, otherwise this thing makes no sense.